### PR TITLE
Making certain affordance-related property chains more specific to av…

### PR DESCRIPTION
**Pull request by mpomarlan**
…oid inconsistencies in plan definitions.

### CODE REVIEW

**Comment by mpomarlan** [comment]
Should address issue #67

**Comment by daniel86** [comment]
**All checks have passed** -- yey :)

I'll have a look over the weekend, thanks for your effort!

File: owl/EASE-OBJ.owl [hunk 1]
```diff
@@ -642,6 +678,18 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#isTaskDefinedByAffordance -->
+
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#isTaskDefinedByAffordance">
```
Comment (daniel86) [inline owl/EASE-OBJ.owl:683]: a comment is missing

File: owl/EASE-OBJ.owl [hunk 2]
```diff
@@ -406,6 +431,17 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#isDefinedByAffordance -->
+
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#isDefinedByAffordance">
```
Comment (daniel86) [inline owl/EASE-OBJ.owl:436]: a comment is missing

Comment (daniel86) [inline owl/EASE-OBJ.owl:436]: shouldn't it be *isDefinedInAffordance* ?

File: owl/EASE-OBJ.owl [hunk 3]
```diff
@@ -26,6 +26,31 @@
...
+        <rdfs:domain rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Affordance"/>
+        <rdfs:range rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Concept"/>
+        <rdfs:label>affordance defines</rdfs:label>
+    </owl:ObjectProperty>
+    
+
+
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#affordanceDefinesTask -->
+
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#affordanceDefinesTask">
```
Comment (daniel86) [inline owl/EASE-OBJ.owl:43]: a comment is missing

File: owl/EASE-OBJ.owl [hunk 4]
```diff
@@ -26,6 +26,31 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#affordanceDefines -->
+
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#affordanceDefines">
```
Comment (daniel86) [inline owl/EASE-OBJ.owl:31]: a comment is missing

**Comment by daniel86** [comment]
sorry for the delay. I only have minor complains:
- there should be a comment for each new relation
- I think there is a typo in one relation name

**Pull request merged**